tests/Unit/Models/VoucherStateModelTest.php

```diff
@@ -101,9 +108,71 @@ public function testADispatchedVoucherCanBeCollected()
         $this->assertEquals($voucher->currentstate, 'recorded');
     }
 
+    /** @test */
+    public function testOnlyADispatchedVoucherCanBeExpiredOrVoided()
+    {
+        Auth::login($this->marketUser);
+        $v = factory(Voucher::class, 'requested')->create();
+        $this->assertEquals($v->currentstate, 'requested');
+
+        // Cant get there from requested
+        $this->assertFalse($v->transitionAllowed("expire"));
```

Comment:
Random mixed use of double and single quotes starting to get on my nerves. ;) We should make some standard rules.